src/engine/source/builder/builders/opBuilderHelperMap.cpp

```diff
@@ -426,15 +426,15 @@ base::Expression opBuilderHelperStringTrim(const std::any& definition)
     if ('\0' == trimType)
     {
         throw std::runtime_error(fmt::format(
-            "\"{}\" function: Invalid trim type \"{}\"", name, parameters[0].m_value));
+            R"("{}" function: Invalid trim type "{}")", name, parameters[0].m_value));
```

Comment:
https://github.com/wazuh/wazuh/wiki/Coding-style-guide-%28C-plus-plus%29#modern-language-features-and-best-practices

Raw strings